kevm-pyk/src/kontrol/foundry.py

```diff
@@ -84,6 +85,14 @@ def profile(self) -> dict[str, Any]:
     def out(self) -> Path:
         return self._root / self.profile.get('out', '')
 
+    @property
+    def proofs_dir(self) -> Path:
+        return self.out / 'apr_proofs'
```

Comment:
```suggestion
    @property
    def proofs_dir(self) -> Path:
        return self.out / 'proofs'
```

Let's just use the word `proofs`, because we'll be eventually storing other types of proofs too.

Reply:
Understood, thank you!